.github/workflows/web_build.yml

```diff
@@ -1,14 +1,20 @@
-###
-name: Automatic web build
+name: "Automatic web build"
+
 on:
   push:
     branches:
-      - master
+      - "master"
+
 jobs:
   build_web_site:
-    runs-on: [ web-builder-22.04 ]
+    runs-on: "ubuntu-latest"
     steps:
-      - name: Build_web
-        run: /scripts/build_web.sh
-      
-
+      - name: "Trigger deployment workflow"
+        run: |
+          gh workflow run deploy-openssl-library-org.yml
```

Comment:
Where is `deploy-openssl-library-org.yml`?

Reply:
it's [here](https://github.com/openssl/openssl-web/blob/main/.github/workflows/deploy-openssl-org.yml)